pkg/features/kube_features.go

```diff
@@ -433,6 +433,12 @@ const (
 	//
 	// Enables the OpenStack Cinder in-tree driver to OpenStack Cinder CSI Driver migration feature.
 	CSIMigrationOpenStack utilfeature.Feature = "CSIMigrationOpenStack"
+
+	// owner: @wojtek-t
+	// alpha: v1.15
+	//
+	// Enables support for watch bookmark events.
+	WatchBookmark utilfeature.Feature = "WatchBookmark"
```

Comment:
@liggitt - where this feature gate should be defined?
We definitely need access to it in apiserver, but we also need it in client-go (e.g. to enable bookmarks in reflector library based on that)?
Thoughts?

Reply:
So thinking more about it, client-go in general shouldn't have access to feature gates.
So maybe this place is fine, because it will allow us to make both
- kube-apiserver set server-sde support based on it
- controllers, kubelets, etc - request bookmarks based on that
WDYT?

Reply:
I am afraid we can not, if we add check the feature gate in reflector, we have to import the feature in client-go, which is what we want to avoid, right?

Reply:
we don't need to import to-client-go, we can properly set ListOptions in those components themselves...

Reply:
this should go in `staging/src/k8s.io/apiserver/pkg/features/kube_features.go`, since watch behavior is part of generic apiserver

you are correct that client-go should not check the feature gate, just the option in ListOptions. see https://github.com/kubernetes/kubernetes/pull/51876/files for how @smarterclayton did this for paging

Reply:
Done. And thanks for the link.

Reply:
The current implementation is already here.

Reply:
Yes I know - but feature gates is kind of api, so I added it to this PR too.